### PR TITLE
DATA-5302: improve docs

### PR DESCRIPTION
**Pull request by aniham**

### CODE REVIEW

File: examples/events/readme.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ OR
...
 | vCTR             | [`page-view`](./page-view.md),[`recs-request-sent`](./recs-request-sent.md), [`recs-response-received`](./recs-response-received.md), [`recs-unit-render`](./recs-unit-render.md), [`recs-unit-view`](./recs-unit-view.md), [`recs-item-click`](./recs-item-click.md), [`recs-add-to-cart-click`](./recs-item-add-to-cart.md) | unitId, sku |
 
+#### Other use cases and applications
+
+A number of Adobe Commerce products utilizing commerce event collection send data additional data that is used for Product Analytics and other internal tracking.
+
+##### Tracking storefront templates
+
+Set [strorefront context](./example-contexts/mock-storefront-context.md) to the [storefront template](../../packages/storefront-events-sdk/src/types/schemas/storefrontInstance.ts) for your store. If your storefront is not yet defined in the Commerce Events schema, you can request for it to be added following the contribution guildelines [here](../../CONTRIBUTING.md).
```
Comment (dobooth) [inline examples/events/readme.md:63]:
```suggestion
Set [strorefront context](./example-contexts/mock-storefront-context.md) to the [storefront template](../../packages/storefront-events-sdk/src/types/schemas/storefrontInstance.ts) for your store. If your storefront is not yet defined in the Commerce Events schema, follow the contribution guildelines [here](../../CONTRIBUTING.md) to have it added.
```

And we can remove 2 of the blank lines below.

File: examples/events/readme.md [hunk 2]
```diff
@@ -54,6 +54,16 @@ OR
 | CTR              | [`page-view`](./page-view.md),[`recs-request-sent`](./recs-request-sent.md), [`recs-response-received`](./recs-response-received.md), [`recs-unit-render`](./recs-unit-render.md), [`recs-item-click`](./recs-item-click.md), [`recs-add-to-cart-click`](./recs-item-add-to-cart.md)                                          | unitId, sku |
 | vCTR             | [`page-view`](./page-view.md),[`recs-request-sent`](./recs-request-sent.md), [`recs-response-received`](./recs-response-received.md), [`recs-unit-render`](./recs-unit-render.md), [`recs-unit-view`](./recs-unit-view.md), [`recs-item-click`](./recs-item-click.md), [`recs-add-to-cart-click`](./recs-item-add-to-cart.md) | unitId, sku |
 
+#### Other use cases and applications
+
+A number of Adobe Commerce products utilizing commerce event collection send data additional data that is used for Product Analytics and other internal tracking.
```
Comment (dobooth) [inline examples/events/readme.md:59]: Is "Product Analytics" an official product? If not, it should be lowercase.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+We love contributions from our community! If you'd like to go a step beyond contributor
+and become a committer with full write access and a say in the project, you must
+be invited to the project. The existing committers employ an internal nomination
+process that must reach lazy consensus (silence is approval) before invitations
+are issued. If you feel you are qualified and want to get more deeply involved,
+feel free to reach out to existing committers to have a conversation about that.
+
+## Security Issues
+
+Security issues shouldn't be reported on this issue tracker. Instead, [file an issue to our security experts](https://helpx.adobe.com/security/alertus.html).
```
Comment (dobooth) [inline CONTRIBUTING.md:47]:
```suggestion
Security issues should not be reported on this issue tracker. Instead, [file an issue to our security experts](https://helpx.adobe.com/security/alertus.html).
```

**Comment by aniham** [comment]
@dobooth thank you for the review!! I've committed your changes and I added a link to Adobe Product Analytics in the readme

https://business.adobe.com/products/product-analytics/adobe-product-analytics.html

**Pull request merged**